packages/pwa-kit-create-app/assets/bootstrap/js/overrides/app/ssr.js

```diff
@@ -38,23 +37,25 @@ const {handler} = runtime.createHandler(options, (app) => {
     // Set HTTP security headers
     app.use(
         helmet({
+            // pwa-kit-runtime ensures that the Content-Security-Policy header always contains the
+            // directives required for PWA Kit to function. Add custom directives here.
             contentSecurityPolicy: {
                 useDefaults: true,
                 directives: {
-                    'img-src': ["'self'", '*.commercecloud.salesforce.com', 'data:'],
-                    'script-src': ["'self'", "'unsafe-eval'", 'storage.googleapis.com'],
-                    'connect-src': ["'self'", 'api.cquotient.com'],
-
-                    // Do not upgrade insecure requests for local development
-                    'upgrade-insecure-requests': isRemote() ? [] : null
+                    scriptSrc: [
+                        // used by the service worker in /worker/main.js
+                        'storage.googleapis.com'
+                    ]
```

Comment:
I'm assuming that we use `eval` somewhere in the SDK and that is why we are blessing it as a required directive?